docs/scaling.md

```diff
@@ -10,8 +10,7 @@ You cannot horizontally scale the controller.
 
 You can scale the controller vertically:
 
-- If you have workflows with many steps, increase `--pod-workers`.
```

Comment:
I don't recommend changing `--pod-workers` anymore. It has no impact in any testing I've done.